Automattic/jetpack
# Slideshow Block: Prevent gap from displayed below slideshow when a number of images are present.

Fixes [#63082](https://github.com/Automattic/wp-calypso/issues/63082)

#### Changes proposed in this Pull Request:
Slideshow block uses css class .swiper-container, however when coblocks plugin is installed it has a a class of similar name which cause the display to be `display:flex` causing a gap when a number of images are in the slideshow.

* Override css class display to always be `display:block`

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






1. Create a JN site 
2. Install the coblock plugin
3. Create a post with a slideshow block and add roughly 40+ images
4. Verify that there's a gap between slideshow block and the content below, also slideshow pagination doesn't show
5. Apply this change
6. Verify there is now no gap and the slideshow pagination now shows.

#### Before
<img width="1127" alt="Markup 2022-05-04 at 09 46 03" src="https://user-images.githubusercontent.com/27249804/166732398-c2f9a91d-b002-409d-a733-2c0c3ecb60c9.png">

#### After
![Screenshot 2022-07-05 at 1 17 27 PM](https://user-images.githubusercontent.com/47489215/177390694-ad2b3bdc-1fe9-47ec-98e3-0a7f593f9d3e.png)
